src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -595,38 +588,33 @@ export default {
 			}
 		}
 
-		const query = gql`query borrowerProfileMeta(
-			$loanId: Int!,
-		) {
-			lend {
-				loan(id: $loanId) {
-					id
-					...on LoanPartner {
-						partnerName
-						partner {
-							id
-							countries {
-								name
+		// Async data fetch for MARS-317
+		const { data } = await this.apollo.query({
```

Comment:
Do we want to wrap this in a try catch?

Reply:
No, `query` rejects the promise it returns when there is an error instead of throwing. I think `readQuery` might be the only ApolloClient function that throws an error (at least the only one we use regularly).

Reply:
Ah I see we've set the `errorPolicy` to `all` in `apollo.js`.